articles/microsites/call-api/call-api-m2m-app.md

```diff
@@ -33,14 +33,13 @@ For M2M applications, Auth0 uses the [M2M Flow](/flows/concepts/m2m-flow).
 ::: steps
   1. <h4>Configure your API</h4>Once you have created your API, you will need to authorize your M2M application and configure any scopes that applications can request during authorization.
 
-  2. <h4>Get an Access Token</h4>Your app requests an Access Token from your Auth0 Authorization Server using the [M2M Flow](/flows/concepts/m2m-flow).
+  2. <h4>Get an Access Token</h4>Your app requests an Access Token from your Auth0 Authorization Server using the <a href="/flows/concepts/m2m-flow">M2M Flow</a>.
 
   3. <h4>Call your API</h4>When your app calls your API, it includes the retrieved Access Token in the HTTP Authorization header.
-
 :::
 
 
-The easiest way to implement the M2M Flow is to [follow our Backend/API Quickstarts](/quickstart/backend).
+To implement a machine-to-machine flow, [follow our Backend/API Quickstarts](/quickstart/backend). The "Calling your API" section shows the required steps.
```

Comment:
This is the only place we call it "a machine-to-machine flow" in the doc, which may be fine. Just pointing it out since it's an outlier.